enhancements/monitoring/alerting-consistency.md

```diff
@@ -123,6 +123,10 @@ these alerts if they are not actionable.
   * Runbook style documentation for resolving critical alerts is required.
     These runbooks are reviewed by OpenShift SREs and currently live in the
     [openshift/runbooks][2] repository.
+* Operator Alerts are RECOMMENDED to include `kubernetes_operator_part_of` label
+  indicating the operator name the alert is related to.
+* Operator Alerts are RECOMMENDED to include `kubernetes_operator_component` label
```

Comment:
I presume we're talking about the operand? If yes, how is it different from the `service` or `job` labels?

Reply:
@simonpasquier I see that we don't set the these labels.  Can you please provide me with how is it being set in other operators?

Also, I didn't see documentation for these labels. I must say that from the name it is not that clear what they exactly mean and also in most cases I see they are set to the same value.